### PR TITLE
fix connection leak

### PR DESCRIPTION
**Pull request by Soniccube**
fix connection leak when making connection.

### CODE REVIEW

**Comment by Nepherhotep** [comment]
Hey @Soniccube, thanks for providing the fix. We will review it and merge asap.

File: aiozk/session.py [hunk 1]
```diff
@@ -84,6 +84,8 @@ def __init__(self, servers, timeout, retry_policy, allow_read_only):
                 conn = await self.make_connection(host, port)
                 if not conn or (conn.start_read_only and not allow_read_only):
                     continue
+                else:
```
Comment (Nepherhotep) [inline aiozk/session.py]: Why are you quitting the loop here?

Comment (Soniccube) [inline aiozk/session.py]: When you pass multi servers into Client class, for example servers="zk1,zk2,zk3", then you will make one connection to each server, but only the last connection you will hold in self.conn, the others are leaking. I am quiting the loop when I get the first available connection.

Comment (Soniccube) [inline aiozk/session.py]: [wglass/zoonado](https://github.com/wglass/zoonado/tree/master/zoonado) has the same mistake.

Comment (cybergrind) [inline aiozk/session.py]: @Soniccube yeah, looks like this is broken. Thank you for your pull request.
Could you please just remove `else` statement and put `break` on next line, like:
```
if something:
    continue
break
```
And I will merge it.

Comment (Soniccube) [inline aiozk/session.py]: @cybergrind updated! thanks.

**Comment by cybergrind** [comment]
@Soniccube thank you

**Pull request merged**